packages/js/src/executors/tsc/tsc.impl.ts

```diff
@@ -74,6 +75,10 @@ export async function* tscExecutor(
     dependencies.push(tsLibDependency);
   }
 
+  const packageJsonFileExists = fileExists(
+    join(context.root, projectRoot, 'package.json')
+  );
+
```

Comment:
Since we are removing the check on line 96, lets inline this fileExists call into the callback function on line 113. Inlining it there will have the benefit of picking up a package.json that has been created while running w/ watch enabled.

Reply:
Done